ansible/configs/sap-integration/custom_workloads.yml

```diff
@@ -0,0 +1,109 @@
+- name: Set Ansible Python interpreter to k8s virtualenv
+  set_fact:
+    ansible_python_interpreter: /opt/virtualenvs/k8s/bin/python
+
+- name: Create openshift-storage namespace
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/ns_storage.j2' ) | from_yaml }}"
+
+- name: Label nodes for OCS
+  # Using shell to support stream operations
+  shell: >
+    for n in $(oc get nodes | grep worker 
+    | awk {'print $1'}); do oc label node $n 
+    cluster.ocs.openshift.io/openshift-storage='';done
+
+- name: Create ocs operatorgroup
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/ocs_operatorgroup.j2' ) | from_yaml }}"
+
+- name: Create ocs subscription
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/ocs_subscription.j2' ) | from_yaml }}"
+
+- name: Give the cluster some time to deploy both operators
+  pause:
+    minutes: 5
```

Comment:
Can we add an actual check in here rather than 5 minutes of hope?

Reply:
If I do that using ansible `wait` for example, I'll need to maintain this going forward to change the `ClusterServiceVersion` objects on every new minor release. I know this is not "ideal" but is the best approach to remove any required maintenance on the config taking in care how often these objects change

Reply:
I do not follow what you mean. You are waiting for an operator(s) to be deployed. Are you expecting different pods to be deployed with new minor updates to the CSV? All I am asking is to watch for the objects using `retries`, `delay`, and `until`.

If you are worried about minor changes the a CSV, then you should look at mirroring and maintaining a copy of the catalog image that has versions you want to support in this config.

https://github.com/redhat-cop/agnosticd/blob/development/docs/Operator_Catalog_Snapshots.adoc